### PR TITLE
Feed and Card Block Changes

### PR DESCRIPTION
**Pull request by jindaliiita**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #<gh-issue-id>

Test URLs:
- Original: https://www.sunstar.com/healthy-thinking/6-things-you-didnt-know-about-your-oral-health
- Before: https://main--sunstar--hlxsites.hlx.live/healthy-thinking/6-things-you-didnt-know-about-your-oral-health
- After: https://feed-changes--sunstar--hlxsites.hlx.live/healthy-thinking/6-things-you-didnt-know-about-your-oral-health

- [ ]  **If you are adding a new block or a variation to an existing block**, please fill below:

  Block library path: https://<branch>--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/<your-block>&index=0

### CODE REVIEW

**Review by dnbute — COMMENTED** [review]
Screenshot 2023-10-05 at 11 53 17
Seems the year is not properly displayed

File: blocks/cards/cards.css [hunk 1]
```diff
@@ -77,17 +78,25 @@ main .block.cards {
...
   color: var(--light-black);
+  padding: 10px 10px 0;
 }
 
 .cards .cards-card-body .title {
   margin-bottom: 1rem;
 }
 
+.cards.large-text .cards-card-body .date {
+  font-size: 12px;
```
Comment (bosschaert) [inline blocks/cards/cards.css:92]: There is a variable for this `--body-font-size-xs`

File: blocks/cards/cards.css [hunk 2]
```diff
@@ -77,17 +78,25 @@ main .block.cards {
...
+}
+
+.cards.large-text .cards-card-body .title {
+  font-size: 16px;
```
Comment (bosschaert) [inline blocks/cards/cards.css:96]: There is a variable for this

File: blocks/cards/cards.css [hunk 3]
```diff
@@ -281,6 +290,14 @@ main .cards.machine-parts ul > li {
     padding: 20px 0 0;
   }
 
+  .cards.large-text .cards-card-body .date {
+    font-size: 14px;
```
Comment (bosschaert) [inline blocks/cards/cards.css:294]: There is a variable for this

File: blocks/cards/cards.css [hunk 4]
```diff
@@ -281,6 +290,14 @@ main .cards.machine-parts ul > li {
...
+  }
+
+  .cards.large-text .cards-card-body .title {
+    font-size: 20px;
```
Comment (bosschaert) [inline blocks/cards/cards.css:298]: There is a variable for this

File: blocks/cards/cards.css (hunk 2 above)
Comment (jindaliiita) [reply blocks/cards/cards.css:96]: Yes, i forgot to check that.
Thanks

File: blocks/cards/cards.css [hunk 5]
```diff
@@ -30,22 +29,21 @@ main .block.cards {
...
 .cards > ul > li:hover {
   box-shadow: 0 4px 1rem 0 rgb(60 70 80 / 40%);
 }
 
 .cards > ul > li .date {
-    font-size: var(--body-font-size-xs);
-    color: var(--transparent-grey-color);
-    font-weight: var(--font-weight-medium);
-    padding-bottom: 0.5rem;
+  font-size: var(--body-font-size-xs);
```
Comment (sdmcraft) [inline blocks/cards/cards.css:44]: I think you'd still need to retain the color. The date is lighter in shade on the reference site.

File: blocks/cards/cards.css [hunk 6]
```diff
@@ -77,17 +75,25 @@ main .block.cards {
...
   text-align: left;
   color: var(--light-black);
+  padding: 10px 10px 0;
 }
 
 .cards .cards-card-body .title {
   margin-bottom: 1rem;
 }
 
+.cards.large-text .cards-card-body .date {
```
Comment (sdmcraft) [inline blocks/cards/cards.css:88]: Do you see cards where `large-text` may not be needed? If no, we can make this the default style for all the cards.

File: blocks/cards/cards.css [hunk 7]
```diff
@@ -77,17 +75,25 @@ main .block.cards {
...
+  font-size: var(--body-font-size-xs);
+}
+
+.cards.large-text .cards-card-body .title {
```
Comment (sdmcraft) [inline blocks/cards/cards.css:92]: Same as above about adding `large-text` variant.

File: styles/styles.css [hunk 8]
```diff
@@ -716,6 +716,10 @@ main .section.light-blue {
   background-color: var(--transparent-blue-light-color);
 }
 
+main .section.no-padding .section-container {
+  padding: 0;
```
Comment (sdmcraft) [inline styles/styles.css:720]: Is this needed? I think all `section-containers` must have same styling unless there's a special case to handle.

File: blocks/cards/cards.css (hunk 5 above)
Comment (jindaliiita) [reply blocks/cards/cards.css:44]: Its for accessibility

File: blocks/cards/cards.css (hunk 6 above)
Comment (jindaliiita) [reply blocks/cards/cards.css:88]: Yes for featured articles font-size is less

File: styles/styles.css (hunk 8 above)
Comment (jindaliiita) [reply styles/styles.css:720]: Removed

Comment (jindaliiita) [reply styles/styles.css]: Needed for tablet view (cards block)

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**